### ISSUE TITLE
non Asian languages

### ISSUE THREAD

**Issue by alain2208** [ttsunion/Deep-Express#3]
Can you please decribe what should be changed to use non Asian Language?

**Comment by alain2208** [comment]
In fact I found it myself. replace in 
parameters.py vocab_size = 28 i think 26 + % + space is that right?
in text.py replace string by latin alphabet plus space plus%

**Comment by FonzieTree** [comment]
@alain2208 Yeah, you are right, have a try, looking forward to know your result.

**Comment by FonzieTree** [comment]
@alain2208 , please download the latest version, noise problem almost disappear in this demo.

**Comment by alain2208** [comment]
i made a personal record and increase the level got very good quality.
As your test are in Chinese I think (sorry if it's not chinese) it's not very easy for non chinese to work.
How is it suppose to work It loads 4 files but only try to reproduce the first one that's right?
Nice job! very impressed to hear my voice back by a computer. even after 4000 run already got very good result.

**Comment by FonzieTree** [comment]
Thank you for your feedback.

**Comment by edwinlimlx** [comment]
@alain2208 as mentioned the following change for EN language
vocab_size = 28
text.py replace string by latin alphabet plus space plus%

does that mean your samples/labels are just abc...?
how can we make it speak via words?
